requirements/text.txt

```diff
@@ -1,2 +1,3 @@
 nltk>=3.6
-bert-score==0.3.10
+transformers>=4.0
```

Comment:
can we have it as another optional? I guess the main point is to drop dependency on HF (not that I don't like nut lets try to be as light as possible - using PT only)